resources/shaders/tile_frag.glsl

```diff
@@ -0,0 +1,26 @@
+#version 330
+
+uniform mat4 modelViewMatrix;
+
+in vec3 fragPosition;
+in vec3 fragNormal;
+
+out vec4 finalColor;
+
+const vec3 lightIntensity = vec3(1.0, 1.0, 1.0);
+const vec3 ambientLight = vec3(.0, .0, .0);
+const vec3 lightPosition = vec3(0., 1.0, 0.0);
```

Comment:
This is a directional light, since it is not put into eye coordinates (i.e. affected by `modelViewMatrix`). It'll be something to test in the future, but I think for now it should work for our use cases.